minethd.cpp

```diff
@@ -33,7 +33,11 @@
 
 void thd_setaffinity(std::thread::native_handle_type h, uint64_t cpu_id)
 {
+#ifdef __MINGW64__
+	SetThreadAffinityMask(&h, 1ULL << cpu_id);
```

Comment:
One more point of confusion - `SetThreadAffinityMask` is not provided by the compiler, it is part of the Windows kernel, it is called the same way whether you are using MSVC/GCC/VB/FORTRAN etc.

`std::thread::native_handle_type` on the other hand is a C++ wrapper around HANDLE that can be specific to the particular C++ library.